lib/ssl/test/ssl_dist_SUITE.erl

```diff
@@ -58,7 +58,9 @@
          verify_fun_fail/0,
          verify_fun_fail/1,
          verify_fun_pass/0,
-         verify_fun_pass/1
+         verify_fun_pass/1,
+         epmd_module/0,
```

Comment:
any reason to not add epmd_module testcase to all?
its not executed when whole SUITE is run

Reply:
I forgot! Will fix!